src/ControlSystem/TimescaleTuner.hpp

```diff
@@ -151,7 +151,7 @@ class TimescaleTuner {
 
   DataVector timescale_;
   bool timescales_have_been_set_{false};
-  double initial_timescale_{std::numeric_limits<double>::signaling_NaN()};
+  double initial_timescale_;
```

Comment:
I don't understand what this commit is doing.

Reply:
If you default constructed a TimescaleTuner and then compared it with `==`, you'd get an FPE because of the signaling NaN.

Reply:
So now you're comparing uninintialized memory instead?

Reply:
I'll set it to 0